doc/src/modules/physics/mechanics/examples/four_bar_linkage_example.rst

```diff
@@ -80,3 +81,67 @@ using :class:`~.KanesMethod` as the backend. ::
    >>> simplify(system.form_eoms())
     Matrix([[l2*rho*(-2*l2**2*sin(q3)*u1' + 3*l2*l3*u1**2*sin(q2 + q3)*sin(q2) + 3*l2*l3*sin(q2)*cos(q2 + q3)*u1' - 3*l2*l3*sin(q3)*u1' + 3*l2*l4*u1**2*sin(q2 + q3)*sin(q2) + 3*l2*l4*sin(q2)*cos(q2 + q3)*u1' + 3*l3**2*u1**2*sin(q2)*sin(q3) + 6*l3**2*u1*u2*sin(q2)*sin(q3) + 3*l3**2*u2**2*sin(q2)*sin(q3) + 2*l3**2*sin(q2)*cos(q3)*u1' + 2*l3**2*sin(q2)*cos(q3)*u2' - l3**2*sin(q3)*cos(q2)*u1' - l3**2*sin(q3)*cos(q2)*u2' + 3*l3*l4*u1**2*sin(q2)*sin(q3) + 6*l3*l4*u1*u2*sin(q2)*sin(q3) + 3*l3*l4*u2**2*sin(q2)*sin(q3) + 3*l3*l4*sin(q2)*cos(q3)*u1' + 3*l3*l4*sin(q2)*cos(q3)*u2' + l4**2*sin(q2)*u1' + l4**2*sin(q2)*u2' + l4**2*sin(q2)*u3')/(6*sin(q3))]])
 
+Revealing noncontributing forces
+--------------------------------
+
+To reveal the noncontributing forces at the closing joint, we require to
+introduce auxiliary speeds in the x and y-direction at the endpoint.
+
+   >>> uaux1, uaux2 = dynamicsymbols('uaux1:3')
+   >>> end_point_aux = end_point.locatenew('end_point_aux', 0)
+   >>> end_point_aux.set_vel(N, end_point.vel(N) + uaux1*N.x + uaux2*N.y)
+
+To ensure that speeds are included in the velocity constraints, we must manually
+overwrite the velocity constraints because those are by default specified as the
+time derivatives of the holonomic constraints.
+
+   >>> system.velocity_constraints = [
+   ...    end_point_aux.vel(N).dot(N.x), end_point_aux.vel(N).dot(N.y)]
+
+When adding the noncontributing forces we need them to depend only on the
+auxiliary velocity and not the velocity that is eliminated by the constraints.
+This can be achieved by applying an equal and opposite force to the
+non-auxiliary endpoint.
+
+   >>> faux1, faux2 = dynamicsymbols('faux1:3')
+   >>> noncontributing_forces = [
+   ...   Force(end_point_aux, faux1*N.x + faux2*N.y),
+   ...   Force(end_point, -(faux1*N.x + faux2*N.y)),
+   ... ]
+
+Alternatively, we can specify a new point that already subtracts the velocity
+eliminated by the constraints.
+
+   >>> end_point_forces = end_point.locatenew('end_point_forces', 0)
+   >>> end_point_forces.set_vel(N, uaux1*N.x + uaux2*N.y)
+   >>> noncontributing_forces = [Force(end_point_forces, faux1*N.x + faux2*N.y)]
+
+Next, we can add the auxiliary speeds and noncontributing forces to the system.
+
+   >>> system.add_loads(*noncontributing_forces)
+   >>> system.u_aux = [uaux1, uaux2]
+
+To include the gravity we can use :meth:`~.System.apply_uniform_gravity` before
```

Comment:
```suggestion
To include gravity we can use :meth:`~.System.apply_uniform_gravity` before
```

Reply:
Addressed in 708248d96ca4879663334e43a2631a234c1bd997